frontend/src/components/Milestones.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  faCalendar,
+  faFolderOpen,
+  faFire,
+  faCircleCheck,
+  faCircleExclamation,
+} from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { useRouter } from 'next/navigation'
+import React from 'react'
+import { ProjectMilestonesType } from 'types/project'
+import { formatDate } from 'utils/dateFormatter'
+import AnchorTitle from './AnchorTitle'
+import ItemCardList from './ItemCardList'
+import { TruncatedText } from './TruncatedText'
+
+interface ProjectMilestonesProps {
+  data: ProjectMilestonesType[]
+  showAvatar?: boolean
+  openMilestones?: boolean
+}
+
+const Milestones: React.FC<ProjectMilestonesProps> = ({
+  data,
+  showAvatar = true,
+  openMilestones,
+}) => {
+  const router = useRouter()
+
+  return (
+    <ItemCardList
+      title={
+        <div className="flex items-center gap-2">
+          <AnchorTitle
+            title={openMilestones ? 'Open Milestones' : 'Closed Milestones'}
+            className="flex items-center leading-none"
+          />
+        </div>
+      }
+      data={data}
+      showAvatar={showAvatar}
+      icon={faFire}
```

Comment:
Can we change this to `signs-post` to stay closer to GitHub icon for Milestones?